expression/aggregation.go

```diff
@@ -31,17 +31,26 @@ type AggregationFunction interface {
 	// Update during executing.
 	Update(row []types.Datum, groupKey []byte, ctx context.Context) error
 
+	// StreamUpdate updates data using streaming algo.
+	StreamUpdate(row []types.Datum, groupKey []byte, ctx context.Context) (bool, error)
```

Comment:
I think this interface can be remove. Most of the operations of `Update` and `StreamUpdate` are the same. Only the number of return value is different.